### DISCUSSION TITLE
Welcome to P10-test-project Discussions!

### DISCUSSION THREAD

**Discussion by BekzodKhudayberdiev** [BekzodKhudayberdiev/P10-test-project#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by UmirzakovOzodbek** [comment]


**Reply by UmirzakovOzodbek** [reply]
https://github.com/UmirzakovOzodbek/P10-test-project

**Comment by Devdavlat** [comment]
exceptionlarni yeg'ishda data timeni formatini qaday ko'rinishda yozish kerak edi  ?

**Reply by BekzodKhudayberdiev** [reply]
> exceptionlarni yeg'ishda data timeni formatini qaday ko'rinishda yozish kerak edi ?

dd-mm-YYYY HH:MM

**Comment by shoxruxsunnatov** [comment]
[Countries of the world.cvs ](https://github.com/XudayberdiyevB/P10-test-project/raw/main/csv/countries%20of%20the%20world.csv) faylida xato bor ekan shekilli. Qatorlarni vergul(,) bilan split qilganda xatolik bo'lyapti. Chunki faylda haqiqiy sonlar ham , bilan yozilgan ekan.

**Reply by BekzodKhudayberdiev** [reply]
> [Countries of the world.cvs ](https://github.com/XudayberdiyevB/P10-test-project/raw/main/csv/countries%20of%20the%20world.csv) faylida xato bor ekan shekilli. Qatorlarni vergul(,) bilan split qilganda xatolik bo'lyapti. Chunki faylda haqiqiy sonlar ham , bilan yozilgan ekan.

xatolik yuboring

**Comment by BunyodNaimov** [comment]
https://github.com/BunyodNaimov/Python/tree/main/modul_2/lesson_10

**Comment by Farruxbek2004** [comment]
yunusboyev farruxbek
https://github.com/Farruxbek2004/Farruxbek/tree/main/working_with_files

**Comment by ierkinov16** [comment]
https://github.com/Saidahror13/Python

**Comment by saydulloooo** [comment]
**Tukhtasinov Saydullo**

https://github.com/Tukhtasinov-Saydullo/PythonCourse-2023

**Comment by mxcvllx** [comment]
https://github.com/Odiljon11/P10-test-project 
Toirjonov Odiljon

**Comment by saydulloooo** [comment]
**Tukhtasinov Saydullo**
https://github.com/Tukhtasinov-Saydullo/PythonCourse-2023

**Comment by shoxruxsunnatov** [comment]
https://github.com/shoxruxsunnatov/solution_for_csv_issues

**Comment by BunyodNaimov** [comment]
Hello Python

**Comment by Devdavlat** [comment]
https://github.com/Devdavlat/test_projects_p10/tree/main/main_homeworks_10

**Comment by BunyodNaimov** [comment]
Vazifalani tashlab yuboring

**Comment by Devdavlat** [comment]
ustoz repo larimdagi malumotlarni o'zgartirib push qisam "Push main to origin/main was rejected by the remote"  shunqa yozuv chiqyabdi remoteni qanday o;zgartirish kerek

**Reply by BekzodKhudayberdiev** [reply]
`git remote -v `
Shu komandani berib ko'ringchi, natijani yuboring

**Reply by Devdavlat** [reply]
$ git remote -v
origin  https://github.com/Devdavlat/test_projects_p10.git (fetch)
origin  https://github.com/Devdavlat/test_projects_p10.git (push)

**Reply by BekzodKhudayberdiev** [reply]
`git pull` qilib keyin push qilib ko'ring, agar yana shu xatolik bo'lsa `git push -f origin main`, lekin main ga push qilmasdan boshqa branchda yuborsangiz yaxshi bo'lardi

**Comment by shoxruxsunnatov** [comment]
@XudayberdiyevB Agar vazifalar boʻlsa tashavoring iltimos.

**Comment by BekzodKhudayberdiev** [comment]
Vazifalarni bugun kechroq yuboraman, bizada hozir svet keldi

**Comment by oktamov** [comment]
https://github.com/oktamov/python_darslar/tree/ozgarish/csv%20vazifa

**Comment by shoxruxsunnatov** [comment]
https://github.com/shoxruxsunnatov/solution-for-Netflix-and-Covid19-data.git

**Comment by Zohirshox** [comment]
https://github.com/Zohirshox/P-10

**Comment by mxcvllx** [comment]
https://github.com/Odiljon11/P10-test-project

**Comment by BunyodNaimov** [comment]
![image](https://user-images.githubusercontent.com/122611882/215728016-4ab8a3dc-c09d-4e6b-8b8d-023c0ccca3a6.png)
.get() nima vazifa bajaryapti? keyni berish keremasmidi?

**Reply by BekzodKhudayberdiev** [reply]
> ![image](https://user-images.githubusercontent.com/122611882/215728016-4ab8a3dc-c09d-4e6b-8b8d-023c0ccca3a6.png) .get() nima vazifa bajaryapti? keyni berish keremasmidi?

bu tkinter Entry yoki boshqa klass metodi, ichiga yozilgan data ni olib berish uchun
